Kconfig

```diff
@@ -9,6 +9,8 @@ mainmenu "RIOT Configuration"
 # For now, get used modules as macros from this file (see kconfig.mk)
 osource "$(KCONFIG_GENERATED_DEPENDENCIES)"
 
+rsource "sys/Kconfig"
```

Comment:
is that supposed to be `osource`?

Reply:
Not in this case. `osource` is for 'optional includes', so the pattern may not match any file. In this case `rsource` is for including files using a relative path.

All the options are documented [here](https://github.com/ulfalizer/Kconfiglib#kconfig-extensions).